### DISCUSSION TITLE
Kook组件 角色相关功能请求

### DISCUSSION THREAD

**Discussion by HeilantG** [simple-robot/Discussions#100] [特性请求]
### 涉及的编程语言

Java

### 涉及的组件库

kook

### 疑问描述

希望实现下列功能

1. 获取服务器角色
2. 获取用户角色
3. 赋予用户角色
4. 删除用户角色

kook相关接口开发文档地址: https://developer.kookapp.cn/doc/http/guild-role

**Comment by ForteScarlet** [comment]
订阅 https://github.com/simple-robot/simpler-robot/issues/618 以跟踪进展
